platform/ios/Makefile

```diff
@@ -9,6 +9,12 @@ appstore_api_issuer_id = $(POLYPOD_APPLE_IOS_APPSTORE_API_ISSUER_ID)
 provisioning_profile_dir := ~/Library/MobileDevice/Provisioning\ Profiles
 provisioning_profile_file := cooppolypolypolypod__iOS_App_Store_Distribution.mobileprovision
 
+build_version := $(shell /usr/libexec/PlistBuddy -c "Print :ApplicationProperties:CFBundleShortVersionString" "${archive_path}/Info.plist")
+build_number := $(shell /usr/libexec/PlistBuddy -c "Print :ApplicationProperties:CFBundleVersion" "${archive_path}/Info.plist")
+appstore_app_id := 1562456558
+beta_testers_group_ids := e03b0ddc-305d-40d5-8186-4dcc32250054
```

Comment:
These should probably into secrets :D

Reply:
those are PolyPod specific. Sure we can add those to secrets as for means of configuration, not sure that exposing those is a security concern per-se :)

Reply:
I think that a great benefit of having those as GH secrets is that changing the the **group ids** will not require a PR(at the same time could be a downside)

Reply:
Yeah, the downside is we can't actually **read** it, only replace it :D Could be incredibly error prone. So I'm leaning a bit towards hard coding the group IDs here, indeed... Might as well hard code both then.

Reply:
As long as we hard code them, maybe do it in a separate file? A dotenv, for instance?

Reply:
Removed the ID, and used the bundle identifier from the exported archive. The group id remains in makefile, most probably this will not change at all, as we will have only one group for automatic releasing. The specific group release will be done by hand when needed.